themes/dark-and-light/v1/theme.json

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema": "https://raw.githubusercontent.com/streetwriters/notesnook-themes/main/schemas/v1.schema.json",
+  "name": "DARK & light",
+  "id": "dark-and-light",
+  "version": 1,
+  "license": "GPL-3.0-or-later",
+  "homepage": "https://notesnook.com",
+  "description": "Dark theme with light everything ⛔ONLY FOR WEB/DESKTOP APP. MAY HAVE SOME ISSUES IN SMARTPHONE APP⛔",
+  "colorScheme": "dark",
+  "compatibilityVersion": 1,
+  "authors": [
+    {
+      "name": "MONARCH",
+      "email": "N/A",
+      "url": "N/A"
```

Comment:
You can omit these.

Reply:
Yep. I did it. Could you please check it again.